packages/images/src/main.test.ts

```diff
@@ -0,0 +1,345 @@
+
+        const match = imageHandler.match(new Request(url))
+
+        expect(match).toBeDefined()
+
+        const response = await match!.handle()
+
+        expect(response.ok).toBe(true)
+        expect(response).toMatchObject(mockedIpxResponse)
+      })
+
+      test('matches on `/.netlify/images/', async () => {
+        const imageHandler = new ImageHandler({
+          logger: createMockLogger(),
+          originServerAddress: 'http://localhost:5173',
+        })
+
+        const url = new URL('/.netlify/images/', 'https://netlify.com')
+        url.searchParams.set('url', 'image.png')
+
+        const match = imageHandler.match(new Request(url))
+
+        expect(match).toBeDefined()
+
+        const response = await match!.handle()
+
+        expect(response.ok).toBe(true)
+        expect(response).toMatchObject(mockedIpxResponse)
+      })
+
+      test('does not match on `/.netlify/foo', () => {
+        const imageHandler = new ImageHandler({
+          logger: createMockLogger(),
+        })
+
+        const url = new URL('/.netlify/foo', 'https://netlify.com')
+        url.searchParams.set('url', 'image.png')
+
+        const match = imageHandler.match(new Request(url))
+
+        expect(match).not.toBeDefined()
+      })
+    })
+
+    describe('request methods', () => {
+      test('allows GET requests', async () => {
+        const imageHandler = new ImageHandler({
+          logger: createMockLogger(),
+          originServerAddress: 'http://localhost:5173',
+        })
+
+        const url = new URL('/.netlify/images', 'https://netlify.com')
+        url.searchParams.set('url', 'image.png')
+
+        const match = imageHandler.match(new Request(url))
+
+        expect(match).toBeDefined()
+
+        const response = await match!.handle()
+
+        expect(response.ok).toBe(true)
+        expect(response).toMatchObject(mockedIpxResponse)
+      })
+
+      test('does not allow POST requests', async () => {
+        const imageHandler = new ImageHandler({
+          logger: createMockLogger(),
+        })
+
+        const url = new URL('/.netlify/images', 'https://netlify.com')
+        url.searchParams.set('url', 'image.png')
+
+        const match = imageHandler.match(new Request(url, { method: 'POST' }))
+
+        expect(match).toBeDefined()
+
+        const response = await match!.handle()
+
+        expect(response.ok).toBe(false)
+        expect(await response.text()).toBe('Method Not Allowed')
+      })
+    })
+
+    describe('local images', () => {
+      let originServer: http.Server
+      let originServerAddress: string
+      const LOCAL_IMAGE_PATH = '/local/image.jpg'
+      const LOCAL_IMAGE_WIDTH = 800
+      const LOCAL_IMAGE_HEIGHT = 400
+
+      beforeAll(async () => {
+        ;[originServer, originServerAddress] = await new Promise<[http.Server, string]>((resolve, reject) => {
+          const originServer = http.createServer(function originHandler(req, res) {
```

Comment:
Please use the `HTTPServer` util from the `dev-utils` package like we do in other tests.